src/FeedItem.php

```diff
@@ -79,10 +82,17 @@ public function author(string $author)
 
         return $this;
     }
+    
+    public function image(string $image)
+    {
+        $this->image = $image;
+
+        return $this;
+    }
 
     public function validate()
     {
-        $requiredFields = ['id', 'title', 'updated', 'summary', 'link', 'author'];
+        $requiredFields = ['id', 'title', 'updated', 'summary', 'link', 'author', 'image'];
```

Comment:
I don't think that the image should be required.

Reply:
agreed